src/controller/java/BUILD.gn

```diff
@@ -351,6 +351,40 @@ kotlin_library("chipcluster_test") {
   kotlinc_flags = [ "-Xlint:deprecation" ]
 }
 
+kotlin_library("matter_controller") {
```

Comment:
rename it to matter_kotlin_controller?

Reply:
rename to kotlin_matter_controller to differentiate with the existing one